Example/Glia Sample Integration/ViewModifiers/EngagementButtonStyle.swift

```diff
@@ -0,0 +1,22 @@
+import SwiftUI
+
+struct EngagementButtonStyle: ButtonStyle {
+
+    private let backgroundColor = Color(red: 0.52, green: 0.11, blue: 0.92)
```

Comment:
This color is used two times, so it would be good to have it in a single place. I know that one is for the buttons outside and another one for the widgets. But imagine you are demoing this in D&D and want to say that it's easy to change colors, but in reality you have to change colors in two places. Because we want to convey the message that your current app's color scheme can be easily picked up by the widgets, it makes sense to have them use the same color and just have to change it once.

Reply:
Fixed [af4a31c](https://github.com/salemove/ios-sdk-widgets/commit/af4a31cbdfdff9ae7acfa5eb81c2eaee841033d5)